client/web/src/cody/components/ChatUI/ChatUi.tsx

```diff
@@ -219,31 +219,38 @@ const FeedbackButtons: React.FunctionComponent<FeedbackButtonsProps> = React.mem
     )
 
     return (
-        <div className={classNames('d-flex', styles.feedbackButtonsWrapper)}>
+        <div className={classNames('d-flex align-items-center', styles.feedbackButtonsWrapper)}>
             {feedbackSubmitted ? (
-                <Button title="Feedback submitted." disabled={true} className="ml-1 p-1">
+                <Button title="Feedback submitted." disabled={true} className="p-1">
                     <Icon aria-label="Feedback submitted" svgPath={mdiCheck} />
                 </Button>
             ) : (
-                <>
+                <div className="d-flex">
                     <Button
                         title="Thumbs up"
-                        className="ml-1 p-1"
+                        className="p-1"
                         type="button"
                         onClick={() => onFeedbackBtnSubmit('positive')}
                     >
                         <Icon aria-label="Thumbs up" svgPath={mdiThumbUp} />
                     </Button>
                     <Button
                         title="Thumbs down"
-                        className="ml-1 p-1"
+                        className="p-1"
                         type="button"
                         onClick={() => onFeedbackBtnSubmit('negative')}
                     >
                         <Icon aria-label="Thumbs down" svgPath={mdiThumbDown} />
                     </Button>
-                </>
+                </div>
             )}
+            <Link
+                to="/get-cody"
+                className="d-inline-block w-100 ml-auto text-right font-italic"
+                onClick={() => eventLogger.log('ClickedOnCodyCTA', { location: 'Chat Response' })}
+            >
+                Use commands, autocomplete and more in your IDE
```

Comment:
Lol do we have a standard for [Oxford comma](https://www.grammarly.com/blog/what-is-the-oxford-comma-and-why-do-people-care-so-much-about-it/) vs. no Oxford comma? 😂 I'm team comma, so I always cry on the inside a little when I see this haha.

Separately, maybe:
```suggestion
                Use commands, autocomplete and more in your IDE.
```

Reply:
Hmm I'm not sure our grammar standard on this throughout Sourcegraph's branding haha. @traceyljohnson ? Copy was approved by @danielmarquespt , adding the `.` ✅